src/utils/hoyolab.ts

```diff
@@ -68,19 +69,14 @@ const getWishes = async (
 ): Promise<GachaItem[]> => {
 	const url = 'https://hk4e-api-os.mihoyo.com/gacha_info/api/getGachaLog';
 	const wishHistory: GachaItem[] = [];
-	let latestTimeSaved = '2020-09-28T00:00:00.000Z';
-
+	let latestSavedWish: Partial<Wish> & { id: string } = { id: '0' }; // yeah I know the type is awful please fix
```

Comment:
Since you 're only using id it should overlap with Wish

```suggestion
	let latestSavedWish: { id: string } = { id: '0' };
```